libminifi/src/core/ProcessorMetrics.cpp

```diff
@@ -122,6 +132,14 @@ std::chrono::milliseconds ProcessorMetrics::getLastSessionCommitRuntime() const
   return session_commit_runtime_averager_.getLastValue();
 }
 
+std::optional<size_t> ProcessorMetrics::getTransferredFlowFilesToRelationshipCount(const std::string& relationship) const {
+  std::lock_guard<std::mutex> lock(transferred_relationships_mutex_);
+  if (transferred_relationships_.contains(relationship)) {
+    return transferred_relationships_.at(relationship);
```

Comment:
double lookup
```suggestion
  const auto relationship_it = transferred_relationships_.find(relationship);
  if (relationship_it != std::end(transferred_relationships_)) {
    return *relationship_it;
```

Reply:
Good point, updated in https://github.com/apache/nifi-minifi-cpp/pull/1909/commits/2723bace043147136941c68f86216fcd61d74a36